go/extra/stats/README.md

```diff
@@ -7,7 +7,7 @@ signature counts.
 
 ```
 $ stats/stats entity-signatures \
-    --address unix:<node_dir>/internal.sock \
+    --address unix:<abstract_socket> \
```

Comment:
Let's not start using abstract sockets everywhere. They should be for E2E tests ONLY, gated behind a don't blame oasis flag.